components/camel-arangodb/src/main/java/org/apache/camel/component/arangodb/ArangoDbComponent.java

```diff
@@ -31,6 +32,8 @@ public class ArangoDbComponent extends DefaultComponent {
 
     @Metadata(label = "advanced", autowired = true)
     private ArangoDB arangoDB;
+    @Metadata(label = "advanced")
```

Comment:
I think it should be autowired = true, then its automatic uses the one from Quarkus out of the box